src/statusBar/statusBar.ts

```diff
@@ -39,15 +39,27 @@ export class StatusBar {
         const uimodeString: string = (workspace.getConfiguration("fileAccess").get("uiMode", "complete"));
         const uimode: UIMode = uimodeString === "complete" ? UIMode.Complete : UIMode.Simple;
         const readOnly = fileAccess ? fileAccess === FileAccess.ReadOnly : Operations.isReadOnly(activeDocument);
+        const colorConfigString: string = (workspace.getConfiguration("fileAccess").get(readOnly ? "readonlyColor" : "writableColor", "none"));
+        const colorString: string = colorConfigString === "none" || colorConfigString === "" ? "" : colorConfigString;
```

Comment:
Replace to use `contributes.Color` as described in https://code.visualstudio.com/api/references/contribution-points#contributes.colors